exercise2/index.php

```diff
@@ -0,0 +1,67 @@
+<?php
+declare(strict_types=1);
+
+spl_autoload_register(function ($class) {
+    $prefix = 'exercise2\\';
+    $baseDir = __DIR__ . '/src/';
+    $len = strlen($prefix);
+
+    if (strncmp($prefix, $class, $len) !== 0) {
+        return;
+    }
+
+    $relativeClass = substr($class, $len);
+    $file = $baseDir . str_replace('\\', '/', $relativeClass) . '.php';
+
+    if (file_exists($file)) {
+        require $file;
+    }
+});
+
+use exercise2\Controller\CalculatorController;
+use exercise2\Model\CalculatorModel;
+use exercise2\Views\CalculatorView;
+
+$model = new CalculatorModel();
+$controller = new CalculatorController($model);
+$view = new CalculatorView();
+
+$result = null;
+$error = null;
+
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
```

Comment:
The process looks good but too many if-else statements. I think it can be better by using guard clause.

Here is for reference:
https://deviq.com/design-patterns/guard-clause